src/integrations/CustomerIO/browser.js

```diff
@@ -69,9 +76,12 @@ class CustomerIO {
 
   page(rudderElement) {
     logger.debug('in Customer IO page');
-
-    const name = rudderElement.message.name || rudderElement.message.properties.url;
-    window._cio.page(name, rudderElement.message.properties);
+    if (this.sendPageNameInSDK === false) {
+      window._cio.page(rudderElement.message.properties);
```

Comment:
have you validated this in customerio dashboard?

Reply:
Yes